src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+        '401':
+          description: Wrong login
+  '/users/me/authenticate-fresh':
+    post:
+      summary: 'Get new access token'
+      operationId: server.routes.users_me.authenticate_fresh
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  format: email
+                password:
+                  type: string
+                  format: password
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: 'Ok'
+        '401':
+          description: 'Unauthorized'
+
+  '/users/me/authenticate-refresh':
+    post:
+      summary: 'Refresh fresh token'
+      operationId: server.routes.users_me.authenticate_refresh
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: ''
+  '/users/me/logout':
+    post:
+      summary: 'Invalidate current token'
+      operationId: server.routes.users_me.logout
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: 'Logged out'
+  '/users/me/password':
+    patch:
+      summary: 'Change password'
+      operationId: server.routes.users_me.change_password
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                password:
+                  type: string
+                  format: password
+                new_password:
+                  type: string
+                  format: password
+                new_password_confirmation:
+                  type: string
+                  format: password
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: 'Ok'
+        '401':
+          description: 'Unauthorized'
+
+  '/users/me/tokens':
+    get:
+      summary: 'List users API tokens'
+      operationId: server.routes.users_me.list_api_tokens
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: 'List of tokens'
+    post:
+      summary: 'Create API token'
+      operationId: server.routes.users_me.create_api_token
+      tags:
+        - users-me
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                name:
+                  type: string
+      responses:
+        '200':
+          description: 'OK'
+  '/users/me/tokens/{jti}':
+    delete:
+      summary: 'Create API token'
+      operationId: server.routes.users_me.revoke_api_token
+      tags:
+        - users-me
+      parameters:
+        - $ref: '#/components/parameters/jti'
+      responses:
+        '204':
+          description: 'Revoked'
+
+
+
+  '/organizations/{org_uuid}/users':
+    post:
+      summary: Add user to organization.
+      description: If user exists, he is added to the org. If he does not, an account is created and he receives an email
```

Comment:
If the users exists, add him to the organization. Otherwise, new account is created and invitiation is sent on his email.